aws/src/integration/java/org/apache/iceberg/aws/lakeformation/TestLakeFormationMetadataOperations.java

```diff
@@ -165,20 +165,20 @@ public void testCreateTableSuccess() {
           schema, partitionSpec, tableLocation, null);
     } finally {
       grantTablePrivileges(testDbName, testTableName, Permission.DELETE, Permission.DROP);
-      glueCatalogPrivilegedRole.dropTable(TableIdentifier.of(testDbName, testTableName), false);
+      glueCatalogPrivilegedRole.dropTable(TableIdentifier.of(testDbName, testTableName), true);
       lfRegisterPathRoleDeleteDb(testDbName);
     }
   }
 
   @Test
-  public void testDropTableSuccessWhenPurgeIsFalse() {
```

Comment:
I think we need test for both `testDropTableSuccess` and `testPurgeTableSuccess`